### DISCUSSION TITLE
Carousel : How can i slide to specific index ?

### DISCUSSION THREAD

**Discussion by zgramming** [mantinedev/mantine#4597] [Q&A] [answered]
Hello, I'm making a feature like an Instagram story, where when the content from the story is finished it will move on to the next story, more or less the flow is like this:

1. Zeffry
     1.1 Content 1 ✅
     1.2 Content 2✅
     1.3 Content 3 ✅ moved to Ari story
2. Ari
     2.1 Content 1
     2.2 Content 2
     2.3 Content 3

For now I'm using the carousel to move from zeffry => arip, but each story has content and where when the content of a story is finished I want to immediately move to the next story.

Actually this feature exists if you use "withIndicator" but I want a freedom to adjust the movement of the slide. I wish there was a property "currentIndex" where this property can change according to our wishes, but I can't find the same thing in the documentation https://mantine.dev/others/carousel/?t=props

After searching another plugin, more or less like this https://react-slick.neostack.com/docs/example/slick-go-to

How do I handle my case?
Thank You.

## Example illustration of my case using Carousel

See the indicator of top, when the content is finished will move to next story

https://github.com/mantinedev/mantine/assets/38829404/6fc3e118-a033-4669-8dbc-13450c33b147

**Comment by rtivital** [comment, marked as answer]
You can scroll to slide with embla instance –  https://www.embla-carousel.com/api/methods/#scrollto
